ProcessHoloFiles.ps1

```diff
@@ -139,110 +55,101 @@ function Select-OutputExtension {
+        Write-Host "Select configuration file for MOMENTS recording:" -ForegroundColor Cyan
+        $configFileMoments = Get-ConfigFileOption
+    }
 }
-else {
-    $total = ($counter - 1)
+
+# Set the frame skip and input fps
+$frameSkip = Read-Host "Enter frame skip (default 16)"
+if (-not ($frameSkip -match '^[0-9]+$')) { $frameSkip = 16 }
+
+$input_fps = Read-Host "Enter input fps (optional, default camera fps)"
+if (-not ($input_fps -match '^[0-9]+$')) { $input_fps = -1 }
+
+$outputExtension = Select-OutputExtension
+
+# List all .holo files in the selected folder
+$holoFiles = Get-ChildItem -Path $holoFolderPath -Filter *.holo -Recurse
+if (-not $holoFiles) {
+    Write-Host "No .holo files found in the selected folder. Exiting script." -ForegroundColor Red
+    exit
 }
-$counter = 1
-$number_pb = 0
 
-# Confirm action with the user before processing files
-Read-Host "Press Enter to start processing files or Ctrl+C to exit"
+Write-Host "Found $($holoFiles.Count) .holo files in the selected folder." -ForegroundColor Cyan
 
-# Function to run Holovibes in CLI
+# Function to execute Holovibes
 function Execute-Holovibes {
     param (
         [string]$inputFilePath,
         [string]$outputFilePath,
         [string]$frameSkip,
-        [string]$configFile = $null
+        [string]$input_fps,
+        [string]$configFile,
+        [bool]$isMoments
     )
 
     $args = "-i `"$inputFilePath`" -o `"$outputFilePath`""
-    if ($script:input_fps -ne -1)
-    {
-        $args += " -f $script:input_fps"
-    }
-    if ($moments -eq 1)
-    {
+    if ($input_fps -ne -1) { $args += " -f $input_fps" }
+    if ($isMoments) {
         $args += " --moments_record"
-    }
-    else {
-        if ($outputExtension -eq ".mp4")
-        {
-            $args += " --mp4_fps 24"
-        } else {
-            $args += " --frame_skip $frameSkip"
-        }
-    }
-
-    if ($configFile) {
-        $args += " -c `"$configFile`""
-        Write-Host "Processing $(Split-Path -Leaf $inputFilePath) with config $configFile..." -ForegroundColor Yellow
     } else {
-        Write-Host "Processing $(Split-Path -Leaf $inputFilePath) without configuration..." -ForegroundColor Yellow
-    }
-    $args
-    Write-Host "Processing holo files ($counter/$total)" -ForegroundColor Cyan
-    $process = (Start-Process -FilePath $exePath -ArgumentList $args -NoNewWindow -PassThru -Wait)
-    if ($process.ExitCode -ne 0)
-    {
-        Write-Host "Program exited with bad exit code" -ForegroundColor Red
-        $script:number_pb += 1
-    }
-    else {
-        Write-Host "Finished processing $(Split-Path -Leaf $inputFilePath), output saved as $(Split-Path -Leaf $outputFilePath)" -ForegroundColor Green
+        $args += " --frame_skip $frameSkip"
     }
+    if ($configFile) { $args += " -c `"$configFile`"" }
+
+    Write-Host "Executing Holovibes with arguments: $args" -ForegroundColor Yellow
+    Start-Process -FilePath "Holovibes.exe" -ArgumentList $args -NoNewWindow -Wait
```

Comment:
For me the .ps1 cannot run Holovibes.exe as it cannot find it when running it from the Git repository.
It should work in the release format (via the installer), but imo it should also work in development circumstances. (how did you test it ?)

Reply:
I tested it by replacing a ps1 in an existing version.

Reply:
Okay, so it works fine in release mode (that's the major part !)
but it would still be better if it worked in dev mode, in the git repository (maybe add/adapt the code I mentioned on my other comment ?)